qt/python/mantidqt/widgets/plotconfigdialog/curvestabwidget/test/test_markertabwidget_view.py

```diff
@@ -0,0 +1,29 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+
+import unittest
+from mantidqt.utils.qt.testing import start_qapplication
+from mantidqt.widgets.plotconfigdialog.curvestabwidget.markertabwidget.view import MarkerTabWidgetView
+
+
+@start_qapplication
+class SampleTransmissionCalculatorViewTest(unittest.TestCase):
```

Comment:
Thanks for the test. I think the name here might be a copy-and-paste error?

Also, can we add it to the list of tests in [qt/python/CMakeLists.txt](https://github.com/mantidproject/mantid/blob/ba31fc5eea89d3fce2076f1fb5edd1659fd8aa0f/qt/python/CMakeLists.txt#L97) so that it can be run with `ctest` and the build servers?